lib/ProductOpener/Ingredients.pm

```diff
@@ -4214,10 +4215,10 @@ my %prefixes_before_dash = (fr => ['demi', 'saint',],);
 my %ignore_phrases = (
 	de => [
 		'\d\d?\s?%\sFett\si(\.|,)\s?Tr(\.|,)?',    # 45 % Fett i.Tr.
-		"inklusive",
+		'inklusive',
 	],
-	en => ["na|n/a|not applicable",],
```

Comment:
The "not applicable" is needed for some manufacturers that import ingredients list with "not applicable" for some products that don't have ingredients listed (e.g. wine, products that are made of one single ingredient like sugar).
I think we should add it back, with a comment to indicate that it's needed and why.